main.go

```diff
@@ -230,29 +231,32 @@ func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *rea
 		os.Exit(1)
 	}
 
+	configMatchSet := match.GetSet()
```

Comment:
Do we want to call it `configMatchSet`? "set" seems to be about the object type, but we are mostly concerned with the intent of the object, not its implementation, and "config" references where we got the data from, but that seems less important than what the data is for.

Maybe something like `operationExcluder`? I'm not in love with that name, just something naming that is more purpose-focused.

Reply:
Agreed, wasn't happy about `configMatchSet` either. Changed to `operationExcluder`, open to suggestions for naming.

Reply:
Sounds like maybe `processExcluder` per your above name suggestions?

Reply:
updated to `processExcluder`